### PR TITLE
added request hash to cached results

### PR DESCRIPTION
**Pull request by DFSko**
resolves happy-travel/agent-app-project#1414

### CODE REVIEW

File: Api/Models/Availabilities/AvailabilityRequestExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Text.Json;
+
+namespace HappyTravel.Edo.Api.Models.Availabilities;
+
+public static class AvailabilityRequestExtensions
+{
+    public static string ComputeHash(this AvailabilityRequest request)
+    {
+        var bytes = JsonSerializer.SerializeToUtf8Bytes(request);
```
Comment (kirillta) [inline Api/Models/Availabilities/AvailabilityRequestExtensions.cs:10]: Can you use existing [extension](https://github.com/happy-travel/edo/blob/master/Api/Infrastructure/HashGenerator.cs)?

File: Api/Models/Availabilities/AvailabilityRequestExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Text.Json;
+
+namespace HappyTravel.Edo.Api.Models.Availabilities;
```
Comment (kirillta) [inline Api/Models/Availabilities/AvailabilityRequestExtensions.cs:4]: Wrong path for extensions anyway

Comment (DFSko) [reply Api/Models/Availabilities/AvailabilityRequestExtensions.cs]: Fixed

File: Api/Models/Availabilities/AvailabilityRequestExtensions.cs (hunk 1 above)
Comment (DFSko) [reply Api/Models/Availabilities/AvailabilityRequestExtensions.cs]: I guess cryptography is not needed here, it slow

Comment (kirillta) [reply Api/Models/Availabilities/AvailabilityRequestExtensions.cs]: Just keep both in one file then

Comment (DFSko) [reply Api/Models/Availabilities/AvailabilityRequestExtensions.cs]: Fixed

**Review by kirillta — APPROVED** [review]

**Pull request merged**